mungegithub/submit-queue/deployment/docs/configmap.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: docs-sq-config
+data:
+  # basic config options.
+  config.http-cache-dir: /cache/httpcache
+  config.organization: kubernetes
+  config.project: kubernetes.github.io
+  config.pr-mungers: size
+  config.state: open
+  config.token-file: /etc/secret-volume/token
+  gitrepo.repo-dir: /gitrepos
+
+  # test-options feature options.
+  test-options.required-retest-contexts: "\"\""
+
+  # submit-queue options.
```

Comment:
This one only turns on the size munger. We can turn on the assignment munger only after https://github.com/kubernetes/kubernetes.github.io/pull/929 is merged I think.